datastore/longhorn.go

```diff
@@ -540,6 +542,21 @@ func (s *DataStore) CreateDefaultNode(name string) (*longhorn.Node, error) {
 			State: types.NodeStateUp,
 		},
 	}
+	diskCapacity, err := s.GetDiskCapacity(types.DefaultLonghornDirectory)
```

Comment:
Don't put this logic in the datastore. Datastore is dealing with CRUD of the object only. It shouldn't contain the logic to determine the field value of the object.